### ISSUE TITLE
Sensors not working

### ISSUE THREAD

**Issue by GegoSK** [LAB02-Research/HASS.Agent#17]
Firstly: thank you very much for this addon. I was looking something similiar for a while. 

After maybe a week testing, two sensors not working for me: CPU Load and CurrentClockSpeed. 

Another "issue". Auto-creating task in Task manager not worked for me. It asked me to fill the name and password. I do not have windows user account with password. I had to do it manually according this https://www.wintips.org/how-to-start-a-program-at-startup-with-task-scheduler/. Similiar link could be mentioned ;). 

Question: 
I know it is "beta". It is possible to make switch sensors (volume up, volume down, media mute, media play pause, restart, shutdown) not to be like switches. More logical could be slider and buttons or service. Thank you!

**Comment by LAB02-Admin** [comment]
Thank you @GegoSK, glad you like it!

And thank you for being observant; the sensors were indeed broken. I've fixed them (hopefully) in the new version, let me know otherwise.

Good point regarding the task without-password. I'll be removing the scheduled task part in the next update, as I'll also be removing the administrator privileges so it won't be necessary anymore.

About your question; sounds good, I'll look into what's possible.

**Comment by GegoSK** [comment]
@LAB02-Admin - thanks, that was quick. Mentioned sensor now worked. I forgot NetworkSensor, I do not see it in MQTT HA.

**Comment by LAB02-Admin** [comment]
> I forgot NetworkSensor, I do not see it in MQTT HA.

Yep, both network- and storage sensor aren't getting added in HA. Scheduled for next version!

**Comment by LAB02-Admin** [comment]
Released a new version that fixes this bug, could you please test it?

**Comment by GegoSK** [comment]
Looks like it works. So many sensors :) I will test it. Thank you very much.

**Comment by LAB02-Admin** [comment]
Hi @GegoSK. I just published a release that removes the need for a scheduled task. During the update, it'll check if there's a task present, and if there is, it'll remove it and activate the new startup mechanism. 

However, since you manually created the task: if the task isn't called `HASS.Agent`, it won't be deleted and the new startup mechanism won't be activated. In that case, please delete the task manually. Then in the **Configuration** window, go to the **Startup** page, and click `enable start-on-login`.

**Comment by GegoSK** [comment]
@LAB02-Admin HASS Agent was updated, before I read your message ;). Task name was the same and it was deleted automatically. :)
